templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>Can we remove this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Of course we can, but why?</p>

Reply:
This was actually a question, not rhetorical. I saw it was commented, do we want to have it here?

Reply:
Lol gotcha.  Yes it can be removed if you like, it's not essential.  I added it so that users could create their own index column if they want to only examine neighborhoods around certain types of cells (like B cells or tumor cells).  It's there just to let them know it's an option, but I didn't want to leave it uncommented and have people blindly clicking through cells and only to get confused when errors pop up because "tumor" isn't one of their clusters.

Reply:
Sounds good